src/util.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Utility Error Messages
+ */
+export const REQUIRE_NON_NULL_ERROR = "This value cannot be null | undefined.";
+export const REQUIRE_STRING_ERROR = "This value must be a string.";
+export const REQUIRE_UINT8ARRAY_ERROR = "This value must be a Uint8Array.";
+
+/**
+ * @param {any | null | undefined} variable
+ * @returns {boolean}
+ */
+export function REQUIRE_NON_NULL(variable) {
```

Comment:
Heh, should of clarified. The functions should still be `camelCase`, but the strings should be `SCREAMING_SNAKE_CASE` since they're non-changing variables.